freeipa/src/main/resources/freeipa-salt/pillar/freeipa/services.sls

```diff
@@ -0,0 +1,9 @@
+freeipa:
+  services:
```

Comment:
I was thinking about this a little bit more and I was thinking we might want to also have nginx and sssd.

Reply:
added